src/features/klant/KlantAanmaken.vue

```diff
@@ -109,11 +109,20 @@ const contactmomentStore = useContactmomentStore();
 
 const savingKlant = ref(false);
 
+const formWarning = ref("");
+
 const telefoonnummerWarning = computed(() => {
   return customPhoneValidator(formData.value.telefoonnummer);
```

Comment:
ik zie dat dit een bestaande functie is, maar misschien de moeite waard om even kijken of dit met een pattern attribuut op het html element kan? dan heb je geen custom error melding html nodig. als dat lastig is, kunnen we hier ook een apart issue voor maken.

Reply:
@felixcicatt heeft deze validator vorige week geschreven (zie onderstaand).

Ik vind 'm te complex om in een pattern attribute te doen (en daardoor ook niet meer herbruikbaar(?)).

Note: het punt is hier júíst dat de gebruiker mag invullen wat die maar wilt, alleen dat er een non-blocking error message wordt gegenereerd wanneer wij denken dat

Laat maar weten wat je ermee wilt doen 👍🏼 

```
function isValidPhoneNumber(val: string) {
  if (!val) return true; // empty is allowed, handeld by required attribute
  const numberCount = (val.match(/[0-9]/g) ?? []).length;
  const hasOnlyAllowedChars = /^(\+|-| |[0-9])*$/.test(val);
  return numberCount >= 10 && hasOnlyAllowedChars;
}
```

Reply:
besproken is geen validatie maar een waarschuwing.